pkg/stats/index.go

```diff
@@ -51,5 +237,17 @@ func Index(w http.ResponseWriter, r *http.Request) {
 		return profiles[i].Name < profiles[j].Name
 	})
 
-	indexTmpl.Execute(w, profiles)
+	data := pageData{
+		Profiles:      profiles,
+		Formats:       []string{"", "json", "yaml"},
+		ResourceTypes: resourceTypes,
+		Clusters:      clusterSlice,
+		Namespaces:    namespaceSlice,
```

Comment:
I think this is okay for now. It would have to be an extremely large deployment to cause issues with the browser and at that point I am much more concerned about the size of the in-memory snapshots we are maintaining.